libs/designer/src/lib/core/actions/bjsworkflow/connections.ts

```diff
@@ -440,3 +443,19 @@ export function getLegacyConnectionReferenceKey(operationDefinition: any): strin
   }
   return referenceKey;
 }
+
+function getHybridTriggerConnectionReferenceKey(operationDefinition: LogicAppsV2.HybridTriggerConnectionInfo): string {
+  let connectionName: string;
+  if (typeof operationDefinition.host.connection.name === 'string') {
```

Comment:
do you guys have two formats for a hybrid trigger?

Reply:
Removed the string check; only one connection type